.github/workflows/composer.test.yml

```diff
@@ -44,3 +44,6 @@ jobs:
       run: docker-compose exec -T api vendor/bin/phpunit
     - name: Psalm
       run: docker-compose exec -T api vendor/bin/psalm
+
+    - name: Run elasticsearch index deletion integration test
+      run: docker-compose exec -e RUN_PHPUNIT_INTEGRATION_TEST=1 -e ELASTICSEARCH_HOST=elasticsearch.svc:9200 -T api vendor/bin/phpunit tests/Jobs/Integration/ElasticSearchIndexDeleteTest.php
```

Comment:
Similar to https://github.com/wbstack/api/pull/155#discussion_r694860962 it might be nice to be able to run multiple integrationy tests here rather than listing them one by one!

(But we can do that later.)